applications/crossbar/priv/couchdb/schemas/devices.json

```diff
@@ -406,6 +406,11 @@
             },
             "type": "object"
         },
+        "rtcp_mux": {
+            "default": true,
```

Comment:
I would not include a default here; the code should assume 'undefined' means 'false' as well, so nothing changes for people unless they explicitly enable this functionality.